tests/vmi_multus_test.go

```diff
@@ -848,16 +853,33 @@ var _ = Describe("SRIOV", func() {
 
 			// manually configure IP/link on sriov interfaces because there is
 			// no DHCP server to serve the address to the guest
-			configInterface(vmi1, "eth1", "192.168.1.1/24", "#")
-			configInterface(vmi2, "eth1", "192.168.1.2/24", "#")
+			configInterface(vmi1, "eth1", cidrA, "#")
+			configInterface(vmi2, "eth1", cidrB, "#")
 
 			// now check ICMP goes both ways
-			pingVirtualMachine(vmi1, "192.168.1.2", "#")
-			pingVirtualMachine(vmi2, "192.168.1.1", "#")
+			pingVirtualMachine(vmi1, cidrToIP(cidrA), "#")
+			pingVirtualMachine(vmi2, cidrToIP(cidrB), "#")
+		}
+
+		It("should connect to another machine with sriov interface over IPv4", func() {
+			pingThroughSriov("192.168.1.1/24", "192.168.1.2/24")
+		})
+
+		It("should connect to another machine with sriov interface over IPv6", func() {
+			pingThroughSriov("fc00::1/64", "fc00::2/64")
 		})
 	})
 })
 
+func cidrToIP(cidr string) string {
+	ip, _, err := net.ParseCIDR(cidr)
+	if err != nil {
+		panic(err)
```

Comment:
Instead of panic, better to use: Expect(err).ToNot(HaveOccurred(), "some error")

Reply:
Done